app/controllers/instrument_duration_rates_controller.rb

```diff
@@ -47,6 +47,8 @@ def manage
   def init_instrument_duration_rates
     @product = Product.find_by!(url_name: params[:id])
 
+    return unless @product.is_a?(Instrument)
```

Comment:
are we showing this tab for other product types?

Reply:
No, only Instruments

Reply:
So why do we need this check?

Reply:
First, sorry I merged before resolving this comment!

I usually prefer to add those checks in case someone gets to the action by using the URL directly (not going through the buttons in the app).